tests/kw_config_loader_test.sh

```diff
@@ -141,4 +143,84 @@ $PWD/$CONFIG_FILENAME"
   assertTrue "Wrong config file reading order.\n$expected_vs_got" $?
 }
 
+function test_show_variables_completeness()
+{
+  local -A shown_options
+  local -A possible_options
+  local output
+
+  # get all assigned options, including commented ones
+  # remove #'s and ='s to get option names
+  output="$(cat 'etc/kworkflow_template.config')"
+  output="$(echo "$output" | grep -oE '^(#?\w+=?)' | sed -E 's/[#=]//g')"
+
+  for option in $output; do
+    possible_options["$option"]='1'
+  done
+
+  output="$(show_variables 'TEST_MODE' | grep -E '^    ')"
+  output="$(echo "$output" | sed 's/.*(\(\S*\)).*/\1/')"
```

Comment:
I think we can ignore this shellcheck warning. However, it is worth checking if you can simplify...

Reply:
Well, the task here is to get what is between parenthesis in the current line. I think it needs a bit of regex anyway...